fairlearn/metrics_plots/__init__.py

```diff
@@ -0,0 +1,18 @@
+# Copyright (c) Microsoft Corporation and contributors.
+# Licensed under the MIT License.
+
+
+"""Module for visualization metrics in notebooks.
+
+This module contains functions for visualizing metrics within
+notebooks.
+"""
+
+
+from .plot_disparities_in_performance import plot_disparities_in_performance
+from .plot_disparities_in_selection_rate import plot_disparities_in_selection_rate
+
+__all__ = [
+    "plot_disparities_in_performance",
```

Comment:
Relaying some considerations from an earlier conversation with @MiroDudik :

We've been wanting to move away from these names since they were confusing to users according to feedback we received. 

Something like `plot_metric_by_group(metric, y_true, y_pred, *, sensitive_features)` would be more intuitive than the "disparity in predictions" used in the past and generally applicable for other metrics compared to `plot_disparities_in_selection_rate`. I understand if you just want to restrict it to the existing selection rate for this PR, though, especially considering that metrics are getting overhauled in parallel.

For the first one we're actually just plotting over- and underprediction, not any arbitrary performance metric. In theory, this could be done as follows
```
plot_error_rate_by_group(y_true, y_pred, *, sensitive_features, over_under_split=True)
```
where `over_under_split` decides whether it's just error rate or split into over- and underprediction (additional arg not required in this PR). However, over- and underprediction also turned out to be unnecessarily complicated for users, so we're planning to switch to FPR and FNR. The difference is that error rate = over + underprediction, whereas FPR and FNR have a different denominator (positives or negatives). Since they're more widely known and used that's more intuitive for users.

Reply:
Closing this comment since I've taken over this PR and replaced this with `plot_disparities_in_metric`. Naming feedback is still welcome, of course!